tests/vm

```diff
@@ -16,6 +16,11 @@ waitInstanceBooted vm1
 echo "==> Test cleanly stopping a VM"
 lxc stop vm1
 
+# Ensure that we can't launch an aliased VM image as a container
```

Comment:
Could we more simply get the `volatile.base_image` value from `v1`?

Reply:
`lxc config get vm1 volatile.base_image` should give you the full fingerprint of the image that's locally cached as it was used as the base image for `vm1`.